ui_testing/testcases/base_test.py

```diff
@@ -84,4 +108,30 @@ def get_active_tst_unit_with_material_type(self, search, material_type='Raw Mate
             if test_unit_dict['Type'] == search and material_type in test_unit_dict['Material Type']:
                 return test_unit_dict
         return {}
+    
+    # this function generates random order data and then redirectes the user to the order's page
+    def get_random_order_data(self):
```

Comment:
okey

Reply:
i added it here in the first place, because it wasn't fully related to order's function, as it generates the data but by creating records in other entities, so i needed a place close to helper file, so i added it in the base test (same as get active article function)